CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- Future Extensions support
```

Comment:
nit * can you add a pr number at the end? e.g. (#123)

Reply:
Done